tests/php/test-class-amp-cli-validation-command.php

```diff
@@ -8,11 +8,14 @@
 use AmpProject\AmpWP\Option;
 use AmpProject\AmpWP\Tests\Helpers\AssertContainsCompatibility;
```

Comment:
```suggestion
```

This is not needed.

You need to remove the `use` statement for this as well below here, I can't reach it with the PR review tools.